src/Servers/Kestrel/Core/src/Internal/Http/HttpHeaders.Generated.cs

```diff
@@ -6105,7 +6105,7 @@ public unsafe void Append(ReadOnlySpan<byte> name, ReadOnlySpan<byte> value)
                 }
 
                 // We didn't have a previous matching header value, or have already added a header, so get the string for this value.
-                var valueStr = value.GetRequestHeaderStringNonNullCharacters(_useLatin1);
+                var valueStr = value.GetRequestHeaderStringNonNullCharacters(UseLatin1);
```

Comment:
This is a `*.Generated.cs`-file, but I don't see any change in the generator itself. So this change will be overwritten soon.

Maybe I'm missing something...

Reply:
This line https://github.com/dotnet/aspnetcore/blob/3787d7e7f070543cc9368d589a504fa8c4bd4830/src/Servers/Kestrel/shared/KnownHeaders.cs#L989 should be updated.

Reply:
Ohhhhhh, please ignore my comments. It's already there.

> (not so easy to find...)

Led me to this comments. Sorry!